tests/phpunit/api/v3/RelationshipTest.php

```diff
@@ -1477,8 +1477,7 @@ public function testCreateWithLesserPermissions() {
    * this from civicrm_generated.mysql
    */
   private function setUpACLByCheating() {
-    CRM_Core_DAO::executeQuery("INSERT INTO civicrm_acl (name, deny, entity_table, entity_id, operation, object_table, object_id, acl_table, acl_id, is_active) VALUES ('Edit All Contacts', 0, 'civicrm_acl_role', 1, 'Edit', 'civicrm_saved_search', 0, NULL, NULL, 1)");
-    CRM_Core_DAO::executeQuery("INSERT INTO civicrm_acl (name, deny, entity_table, entity_id, operation, object_table, object_id, acl_table, acl_id, is_active) VALUES ('Core ACL',0,'civicrm_acl_role',0,'All','access CiviMail subscribe/unsubscribe pages',NULL,NULL,NULL,1)");
```

Comment:
@demeritcowboy why did you include this? :) object_id surely should be an id not a string right? :)

Reply:
It's null, and I took it straight out of civicrm_generated, but the removed "Core ACL" line is necessary for the test above to work if I remember right. The test may now ALWAYS pass which wasn't the intent.

Reply:
I'm open to doing the test a different way - it might be useful to have the test installs mirror more closely a real install for ALL tests.

Reply:
@demeritcowboy the line I removed is also being removed from real databases.

Reply:
Ok I see. I'll do a followup after since the test isn't completely useless but it's not testing what it was testing and the ACL part of it now is a bit misleading.